src/main/java/UnrecognisedInputCommand.java

```diff
@@ -0,0 +1,7 @@
+public class UnrecognisedInputCommand extends Command {
```

Comment:
You are the first person I see with such a command!!! Why didn't you just throw an exception to deal with unrecognised input? (I am genuinely curious!!)

Reply:
I see, it was just part of my design to abstract out as many commands as possible. Abstraction, together with descriptive names for abstracted units (Methods, Classes, etc) also helps me to understand my own code better when the project gets more developed and demanding.